src/entities/notification.entity.ts

```diff
@@ -0,0 +1,22 @@
+import {
+  ENotificationType,
+  ENotificationStatus,
+} from './../interfaces/notification.interface';
+import { BaseEntity } from 'src/shares';
+import { Column, Entity, ManyToOne } from 'typeorm';
+import { User } from './user.entity';
+
+@Entity('notification')
+export class Notification extends BaseEntity {
+  @Column({ enum: ENotificationStatus, default: ENotificationStatus.NotSeen })
+  status: ENotificationStatus;
```

Comment:
thay vì seen hay not seen -> lưu thời gian đọc.
Nếu đọc rồi thì sẽ có thời gian đọc. 
=> Lưu 1 được 2